# licence: modifies PCSX ReARMed (https://github.com/libretro/pcsx_rearmed), GPL-2.0 (its COPYING file); this patch
# is distributed under the same licence.
# reason: the pin's jni/Android.mk names the wrong path for compat_strl.c under USE_LIBRETRO_VFS=1 (encodings/
# instead of compat/), so the VFS build fails. Build script only, no emulation source touched. Fixed upstream in
# 6f230b1 (PR #926, which also made VFS the default). Applies to pin 6eb679d064a1efe92af79fe9970c0591c8da02d4.
# git apply ignores this preamble.
diff --git a/jni/Android.mk b/jni/Android.mk
index 88ca248db66fe6388117e4f58fb337c760e50a60..0e4f4abcb11d722aefb67946431b0975c6e8cdd7 100644
--- a/jni/Android.mk
+++ b/jni/Android.mk
@@ -118,7 +118,7 @@ ifeq ($(USE_LIBRETRO_VFS),1)
 SOURCES_C += \
              $(LIBRETRO_COMMON)/compat/compat_posix_string.c \
              $(LIBRETRO_COMMON)/compat/fopen_utf8.c \
-             $(LIBRETRO_COMMON)/encodings/compat_strl.c \
+             $(LIBRETRO_COMMON)/compat/compat_strl.c \
              $(LIBRETRO_COMMON)/encodings/encoding_utf.c \
              $(LIBRETRO_COMMON)/file/file_path.c \
              $(LIBRETRO_COMMON)/streams/file_stream.c \
